dashboard.py

```diff
@@ -17,23 +18,33 @@
 async def results(request):
     """Fetch remote results forward on."""
 
+    def _derive_mapping_code(response, mapped_totals):
+        """Consumes a list of dictionaries and returns a mapping of 2 char CC-SS (Country State)
+        keys and total submissions for each."""
+        values = response['values']
+        code = '{0}-{1}'.format(
+            [x['value'].lower() for x in values if x['label'] == 'country_code'][0],
```

Comment:
For PollRuns that are incomplete, this may return an IndexError; need to account for this.